### ISSUE TITLE
Selectonemenu: support for aria-labelledby IE11/ JAWS

### ISSUE THREAD

**Issue by HISeG-ME** [primefaces/primefaces#4601]
## 1) Environment
- PrimeFaces version: 6.2
- Affected browsers: Internet Explorer 11 / Microsoft Edge

## 2) Expected behavior
When using the JAWS Screenreader (tested with JAWS17 and JAWS18) with IE11, the Label which is connected to the Selectonemenu Element (ui-helper-hidden-accessible, Input) should be read when it gets the Focus.

Neither the Labels´ for attribute is recognized, nor the manually applied aria-labelledby attribute.

## 3) Actual behavior
Primefaces Showcase Selectonemenu

JAWS17: when tabbing through the different Variants, the Screenreader reads "Listarea" followed by current Value of Selectonemenu (for example Selectonemenu). When tabbing into the Lazy Selectonemenu, the Label gets read (all others not).

JAWS18: the Screenreader reads the Label Advanced and Lazy.

## 4) Steps to reproduce

Internet Explorer 11

https://www.primefaces.org/showcase/ui/input/oneMenu.xhtml

With the Screenreader JAWS17 and/ or JAWS18, tab through the different Variants of Selectonemenu

**Comment by melloware** [comment]
@HISeG-ME to help me fix this can you paste the current HTML generated and the "expected" HTML you would want generated?   It will help me figure out how to fix this if I can see what the ScreenReader is expecting.

**Comment by melloware** [comment]
Hmmm this looks like a a regression because this was fixed 4 years ago to have p:outputLabel for="" set the aria-labelledBy of the for attribute as long as its an InputHolder which SelectOne is.  Original Commit: https://github.com/primefaces/primefaces/commit/6cca4616d028f3b538e3b0a3b0c1881c7d5b5ec2

**Comment by melloware** [comment]
In fact I am not seeing this work for Calendar or any other component that it used to.    I debugged and so far that piece of code in OutputLabel is being called.  However I am not seeing in reflected in the HTML.  

I am going to debug further.

**Comment by melloware** [comment]
OK I debugged it and submitted a fix.  This was definitely an issue for all InputHolder components not just SelectOneMenu.
